packages/bitcoin/src/coin-selection/coin-selection.ts

```diff
@@ -13,29 +13,21 @@ export interface CoinSelectionOutput {
   address?: string;
 }
 
-export interface CoinSelectionUtxo {
```

Comment:
Are you removing this as it's just annoying and easier to use more generic types?